### PR TITLE
Gtk3 sample gtk demo dialog rb

### PR DESCRIPTION
**Pull request by cedlemo**

### CODE REVIEW

File: gtk3/sample/gtk-demo/dialog.rb [hunk 1]
```diff
@@ -1,137 +1,127 @@
...
+    hbox.pack_start(vbox2, :expand => false, :fill => false, :padding => 0)
+    vbox2.pack_start(button, :expand => false, :fill => false, :padding => 0)
+
+    table = Gtk::Grid.new
+    table.row_spacing = 4
+    table.column_spacing = 4
+    hbox.pack_start(table, :expand => false, :fill => false, :padding => 0)
+
+    label = Gtk::Label.new("_Entry 1", :use_underline => true)
+    table.attach(label, 0, 0, 1, 1)
```
Comment (kou) [inline gtk3/sample/gtk-demo/dialog.rb]: I'll merge this pull request but I add a comment.
We should improve `Gtk::Grid#attach` API for easy to use. For example:

```ruby
def attach(label, options={})
  left   = options[:left]   || 0
  top    = options[:top]    || 0
  width  = options[:width]  || 1
  height = options[:height] || 1
  attach_raw(label, left, top, width, height)
end
```

**Comment by kou** [comment]
Thanks!

File: gtk3/sample/gtk-demo/dialog.rb (hunk 1 above)
Comment (cedlemo) [inline gtk3/sample/gtk-demo/dialog.rb]: @kou 
Tell me if you want me to do it

Comment (kou) [inline gtk3/sample/gtk-demo/dialog.rb]: Thanks. Could you try to do it? I want to see scripts that uses the API to confirm the API is really easy to use.

**Pull request merged**